spec/Classes.tex

```diff
@@ -338,7 +385,7 @@ \subsection{The object Class}
 indirectly.  If no class name appears in the inheritance list, the class derives
 implicitly from \chpl{object}.  Otherwise, a class derives from \chpl{object}
 indirectly through the class or classes it inherits.  A variable of type \chpl{object}
-can hold a reference to an object of any class type. 
+can hold a reference to an object of any class type.
```

Comment:
If we are removing multiple inheritance, we should remove "or classes" in the above.